chpl/chpl-service/src/main/java/gov/healthit/chpl/search/dao/ListingSearchDao.java

```diff
@@ -66,14 +66,14 @@ public List<ListingSearchResult> getListingSearchResults() {
     }
 
     private List<ListingSearchResult> convertToListingSearchResults(List<ListingSearchEntity> entities)
-        throws EntityRetrievalException, DateTimeParseException, NumberFormatException  {
+            throws EntityRetrievalException, DateTimeParseException, NumberFormatException  {
         return entities.stream()
-            .map(rethrowFunction(entity -> buildListingSearchResult(entity)))
-            .collect(Collectors.toList());
+                .map(rethrowFunction(entity -> buildListingSearchResult(entity)))
```

Comment:
I notice a lot of stuff like this, where there are spacing changes around code that wasn't actually changed in a way that's relevant to the work of the feature. Guessing this is because of your new Eclipse and whatever settings that has? I don't feel suuuuper strongly, but it does make the PR harder to read. Either I should change my Eclipse to format like yours (in which case we will have lots of these types of changes in all PRs going forward), or you should change yours to match the spacing formatting of the current (which I don't know if the current is more or less correct than what you have now).